documents/motions/15.2022.1 - Suspensions and other Sanctions.md

```diff
@@ -1,45 +1,61 @@
+   2. in the opinion of the WCA Appeals Committee, the Community member is in breach of the Spirit, Mission, Motions, Policies, Codes and/or Regulations of the WCA, or
+   3. the WCA Appeals Committee considers that the Registered Speedcuber does not fulfil the requirements of eligibility for registration as a Registered Speedcuber, or
+   4. the WCA Appeals Committee considers that the presence of a the Community member is a threat to the safety of other Community members or otherwise disruptive for the community.
+8. Before the WCA Appeals Committee may exercise its suspensive rights under this Motion, the Community Member must have been informed of the active appeal and must have been given the opportunity to provide a statement to the WCA Appeals Committee.
+9. A Registered Speedcuber who has been suspended by the WCA Appeals Committee under this Motion may request for reinstatement by the WCA Appeals Committee, with a motivation of the request.
+10. A Registered Speedcuber who has been suspended by the WCA Appeals Committee under this Motion shall automatically cease to be suspended at the end of the fixed period, or once, in the opinion of the WCA Appeals Committee, the set of circumstances specified has changed or ceases to exist.
 11. The WCA Board shall have the following rights, based on a voting in the WCA Board of Directors with a supermajority:
-    1. to remove or deny accreditation, appointment, or acknowledgement of Staff, Registered Speedcubers, or Partners.
-    2. to impose any sanction that the WCA Board deems to be appropriate on Staff, Registered Speedcubers, or Partners.
+    1. to remove or deny accreditation, appointment, or acknowledgement of Staff, Registered Speedcubers, Regional Organizations, or Partners.
+    2. to impose any sanction that the WCA Board deems to be appropriate on Community members.
+12. Prior to exercising its rights under this Motion the WCA Board should consult with the WCA Integrity Committee (where relevant), the local WCA Delegate(s), and the Regional Organization(s) for the respective area (if they exist).
+13. A Regional Organization shall have the right to suspend a Community member from participation in or attendance of WCA Competitions in their region.
+    1. A Regional Organization may only exercise its suspensive rights when it considers that the presence of a Community member is a threat to the safety of other Community members or otherwise disruptive for the community.
+    2. The Regonal Organization must consult with the WCA Integrity Committee for each individual case prior to exercising this supsensive right.
+14. Suspensions and other sanctions should not be placed where they are not permissible under relevant local laws.
+    1. Suspensions and other sanctions which are not permissible under relevant local laws are invalid. Where a suspension or other sanction is partially impermissable, only the impermissible portion is considered invalid. 
```

Comment:
@Epride / @KerrieJarman per slack discussion, added wording on local law exemption. Intention is to cover off instances where it is not possible to ban someone from attending WCA competitions for certain venues, acknowledging that local law supersedes the sanction in this case (i.e. the right to attend the public place where the venue is without discrimination overrules any WCA sanction placed).